### PR TITLE
[SofaConstraint] CLEAN GenericConstraintSolver

### PR DESCRIPTION
**Pull request by olivier-goury**
Simple cleaning.

______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by damienmarchal** [comment]
[ci-build][with-scene-tests]

File: modules/SofaConstraint/GenericConstraintSolver.cpp [hunk 1]
```diff
@@ -506,7 +492,7 @@ bool GenericConstraintSolver::applyCorrection(const core::ConstraintParams *cPar
 
 	if (displayTime.getValue())
     {
-        sout << " TotalTime " << ((double) timerTotal.getTime() - timeTotal) * timeScale << " ms" << sendl;
+        msg_info() << " TotalTime " << ((double) timerTotal.getTime() - timeTotal) * timeScale << " ms";
```
Comment (epernod) [inline modules/SofaConstraint/GenericConstraintSolver.cpp]: Could you change the double into SReal?

**Comment by epernod** [comment]
[ci-build][with-all-tests]

**Review by epernod — APPROVED** [review]

**Comment by olivier-goury** [comment]
> Hi @olivier-goury I tried to fix the conflicts due to Zhongkai PR, let me know if this is fine.

Hi! Fine by me.

**Comment by olivier-goury** [comment]
> Strange, looking at the dashboard: 1) some timeout appeared and 2) regression tests failing
> could this be related to the PR?

Hello Hugo!
Hum, I don't think so, that would be really strange, this PR is purely about cleaning, it does not change the behaviour of the component.

**Pull request merged**